src/cheri/app-cap-description.adoc

```diff
@@ -1,16 +1,21 @@
 [appendix]
-[#app_cap_description]
-== Anatomy of Capabilities in {cheri_base_ext_name}
+[#app_cap_description, reftext="Default capability encoding"]
+== Default Capability Encoding in {cheri_base_ext_name}
```

Comment:
I confess to not being a fan of "default" as the adjective here, but I have nothing better to offer.

Reply:
To be honest `default` seems to be the least contentious name